tests/test_requirements.txt

```diff
@@ -6,10 +6,10 @@ glueviz~=0.13.3
 h5py~=3.1.0
 ipython~=7.6.1
 matplotlib~=3.3
+mock
```

Comment:
@neutrinoceros Just want to make sure I got the change right so that this conversation can be closed.

Reply:
something went wrong, apparently mock was removed _and added back_.

Reply:
Yeah, I added it back in since I didn't see it in the file, but I thought it needed to be there based on your comment. Did I mess up and misunderstand?